### DISCUSSION TITLE
How to change highlight colors?

### DISCUSSION THREAD

**Discussion by robertocommit** [ggandor/lightspeed.nvim#70] [Q&A]
I have a small question, as you can see the uppercase highlight colours are not fully visible (maybe because of my theme?).

Is there a way for me to change them? For example the white uppercase letter to black uppercase letter.

Thanks so much for this masterpiece!

![image](https://user-images.githubusercontent.com/30242227/139583245-bd80d715-cf80-4700-ae45-f689bc2d4326.png)

**Comment by robertocommit** [comment]
Ok duplicated: https://github.com/ggandor/lightspeed.nvim/issues/33

might be worth to mention it in the docs!

Thanks again

**Comment by robertocommit** [comment]
An advice about the picture above, I am trying to change the light green background and white letter into something more readable, using:

:highlight LightspeedShortcutOverlapped guibg=#f00077 guifg=#ffffff

It does not work, what am I missing?

**Comment by ggandor** [comment]
Those are regular `LightspeedShortcut`s, that should work. (N.B.: Technically, overlapped shortcuts are not possible currently, if I'm not mistaken. It could only occur once we implement [optimizations](https://github.com/ggandor/lightspeed.nvim#-coming-sooner-or-later) that would make the order of the labels non-deterministic.) By the way, it is just an accident that most of them happen to be uppercase labels in this example.

See also: https://github.com/ggandor/lightspeed.nvim#highlight-groups

> Thanks so much for this masterpiece!

Thanks for the kind words!

**Comment by robertocommit** [comment]
Sorry to bother, but I am not able to change that light green into something else:

![image](https://user-images.githubusercontent.com/30242227/139700909-76ba1c57-50d7-4067-bf4d-650a342e9eaa.png)

I am trying with this command but it is not working.

`:highlight LightspeedShortcutOverlapped guibg=#f00077 guifg=#ffffff
`
Moreover, is that command working only during that neovim session or is it permanent?

Thanks!

**Comment by ggandor** [comment]
Use `:highlight LightspeedShortcut ...`, not `LightspeedShortcutOverlapped ...`.

> Moreover, is that command working only during that neovim session or is it permanent?

If you want it to be permanent, then you should put that line somewhere in your init.vim (or init.lua, as `vim.cmd("highlight ...")`, after the colorscheme has been loaded.

BTW, what is this colorscheme you're using?
